### ISSUE TITLE
Siriproxy doesn't work when voiceOver is enabled

### ISSUE THREAD

**Issue by juamonco** [plamoni/SiriProxy#379]
I've tried siriproxy with voiceover accessibility feature enabled, because it would be very useful for me but I found it doesn't work. Has anyone tried?

**Comment by plamoni** [comment]
I haven't tried, but I'll give it a whirl when I get home if I have a
chance.

-Pete

**Comment by juamonco** [comment]
Thanks. I have tried siriproxy with an iphone 4 S and iOS6, it worked without problems when VoiceOver was turned off, but when it was on it didn't capture any word.

**Comment by Deanmv** [comment]
Just tried on iPhone 5, iOS 6.0.1 and worked fine with VoiceOver turned on.

**Comment by juamonco** [comment]
Thanks, maybe it is because I use a buetooth headset. Can you try it? Thanks again.

De: Deanmv [mailto:notifications@github.com] 
Enviado el: viernes, 23 de noviembre de 2012 18:15
Para: plamoni/SiriProxy
CC: juamonco
Asunto: Re: [SiriProxy] Siriproxy doesn't work when voiceOver is enabled (#379)

Just tried on iPhone 5, iOS 6.0.1 and worked fine with VoiceOver turned on.

—
Reply to this email directly or view it on GitHub https://github.com/plamoni/SiriProxy/issues/379#issuecomment-10665430 . 

  https://github.com/notifications/beacon/Jshd8sI44GVrKZBvymxqKMwTUbTxITV-cwcceJXmwvKwRBvvUvvH1KpljI4mAzHt.gif

**Comment by Deanmv** [comment]
Don't have a bluetooth headset to hand i'm afraid so can't test that one.
